dotnet/client/src/Azure.Mobile.Client/Auth/PreauthorizedTokenCredential.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Azure.Core;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Mobile.Client.Auth
+{
+    /// <summary>
+    /// A trivial implementation of the <see cref="TokenCredential"/> that allows the developer to
+    /// update the AccessToken on their own schedule.  This is then included in the Authorization header
+    /// on each request.
+    /// </summary>
+    public class PreauthorizedTokenCredential : TokenCredential
+    {
```

Comment:
Let's add two constructors here. One empty ctor and one that takes in the AccessToken.  This will help people new to the library know that they should pass AccessToken if they have it.